app/src/main/java/org/simple/clinic/security/pin/PinEntryCardView.kt

```diff
@@ -73,11 +72,7 @@ class PinEntryCardView(context: Context, attrs: AttributeSet) : CardView(context
   }
 
   private val events by unsafeLazy {
-    Observable
-        .merge(
-            pinTextChanges(),
-            upstreamUiEvents
-        )
+    pinTextChanges()
```

Comment:
There seems to be an unused import for rx `Observable` when you did this, can you fix that, please?

Reply:
I've fixed that in another PR.